acceptance/suites/tests/authorization/default_rules.rb

```diff
@@ -135,6 +135,13 @@ def report_query(node)
     assert_allowed
   end
 
+  step 'static file content endpoint' do
+    # We'd actually need to perform a commit and use its code-id in order to
+    # get back a 200, but we know that a 400 means we got past authorization
+    curl_authenticated('/puppet/v3/static_file_content/foo/bar?environment=production')
+    assert_allowed(400)
+  end
```

Comment:
:)  yes, exactly what I suggested above.  Although I mistakenly said 404 when I should have said 403.